react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(prop) {
+  return <button>{prop}</button>;
+}
+
+const element = CustomButton('Click Me!');
```

Comment:
In react we never really call our own components, instead, we create react elements with them.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```